### PR TITLE
API: additional parameters for list_alerts and list_devices API calls

### PR DESCRIPTION
**Pull request by dsgagi**
- Add filtering by severity in list_alerts.
- Added ability to filter only active devices (not ignored and not disabled) in list_devices.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by dsgagi** [comment]
Will submit one more change to `list_devices` function.

Let me know what you think.

File: doc/API/Devices.md [hunk 1]
```diff
@@ -768,6 +768,7 @@ Input:
   - order: How to order the output, default is by hostname. Can be prepended by DESC or ASC to change the order.
   - type: can be one of the following to filter or search by:
     - all: All devices
+    - active: Only not ignored and not disabled devices
```
Comment (laf) [inline doc/API/Devices.md]: This should be named device_up to match the macro name we have for alerts.

File: html/includes/api_functions.inc.php [hunk 2]
```diff
@@ -1038,10 +1040,18 @@ function list_alerts()
     $sql = '';
     if (isset($router['id']) && $router['id'] > 0) {
         $alert_id = mres($router['id']);
-        $sql      = 'AND `A`.id=?';
+        $sql      = ' AND `A`.id=?';
         array_push($param, $alert_id);
     }
 
+    if (isset($_GET['severity'])) {
```
Comment (laf) [inline html/includes/api_functions.inc.php]: Change the `$_GET` use to `$vars` then you don't need mres + we sanitize the input.

File: doc/API/Devices.md (hunk 1 above)
Comment (laf) [reply doc/API/Devices.md]: Ok makes sense.

**Comment by dsgagi** [comment]
Will wait on #9077 to be merged, and then make proposed changes.

**Comment by dsgagi** [comment]
laf, in the end you did more than I did :)

Sorry about that. Like I said before, php noob here :)

**Comment by laf** [comment]
All good, thanks for contributing :)

**Pull request merged**